src/puppetlabs/puppetdb/queue.clj

```diff
@@ -250,21 +250,32 @@
                  file-extension))))
 
 (defn cmdref->cmd [q cmdref]
+  "Returns the command associated with cmdref, or logically false if
+  the file is missing (i.e. someone has deleted it)."
```

Comment:
is it actually nil if the command is missing? I think being more specific is better.